### PR TITLE
add event report-en

### PR DESCRIPTION
**Pull request by ogwata**
村上さんに確認してもらった後、WilmとFlorianにレビューしてもらいます。

### CODE REVIEW

File: _posts/2021-11-17-meetup-2021-autumn-report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: Event report of “Vivliostyle User/Dev Meetup Autumn 2021”
+lang: en
+image: /assets/posts/2021-11-17-meetup-2021-autumn-report/2021fall_user_meeting_banner.png
+author:  ogwata
+tags:
+  - event
+---
+<div style="float: right; margin: 0 0 1em 1em;"><img src="/assets/posts/2021-11-17-meetup-2021-autumn-report/2021fall_user_meeting_banner.png" alt="Vivliostyle User / Dev Meetup 2021 Autumn" style="width: 400px; box-shadow: 1px 2px 2.5px 1.5px grey;" /></div>
+
+(The original article is [「Vivliostyle ユーザーと開発者の集い 2021秋」開催報告](https://vivliostyle.org/ja/blog/2021/11/17/meetup-2021-autumn-report/)] in Japanese)
```
Comment (MurakamiShinyu) [inline _posts/2021-11-17-meetup-2021-autumn-report.md:11]:
```suggestion
(The original article is [「Vivliostyle ユーザーと開発者の集い 2021秋」開催報告](https://vivliostyle.org/ja/blog/2021/11/17/meetup-2021-autumn-report/) in Japanese)
```

不要な `]` を削除

File: _posts/2021-11-17-meetup-2021-autumn-report.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+The semi-annual “Vivliostyle User/Dev Meetup Autumn 2021” was held online on January 14. The slides and a summary of comments from the day (by yamasy1549) are available at the following page.
```
Comment (MurakamiShinyu) [inline _posts/2021-11-17-meetup-2021-autumn-report.md:13]:
```suggestion
The semi-annual “Vivliostyle User/Dev Meetup Autumn 2021” was held online on November 14. The slides and a summary of comments from the day (by yamasy1549) are available at the following page.
```

**Review by MurakamiShinyu — CHANGES_REQUESTED** [review]

**Review by MurakamiShinyu — APPROVED** [review]

File: _posts/2021-11-17-meetup-2021-autumn-report.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+     - [Q&A session<i class="fas fa-external-link-alt"></i>](https://youtu.be/z31H5T23-fA) (14:27)
+- [Part 4: Vivliostyle Dev Meeting<i class="fas fa-external-link-alt"></i>](https://youtu.be/bQRZlymV3-8) (46:48)
+
+This time, the start time was delayed by about 30 minutes due to my poor setup. Also, due to this, I was not able to get the questionnaire that I had planned for Ayumu Takai's presentation, which left me with many things to regret. We deeply apologize for the inconvenience caused to everyone who applied for participation and the speakers. 
+
+I was impressed by Zaslon's point that "the information on the official website of Vivliostyle is only the explanation of the application, not the CSS typesetting itself". Certainly, from the user's point of view, the information on the official website seems to be biased. Just a few days ago, I had a conversation with Founder Murakami saying, “We are achieving the initial development goals for Vivliostyle.js, so let's work on the spread of CSS typesetting as the next step”. It's a good opportunity, so let's think about what we can do.
```
Comment (johanneswilm) [inline _posts/2021-11-17-meetup-2021-autumn-report.md:37]: Maybe change this:

> ...the information on the official website of Vivliostyle is only the explanation of the application...

to 

> ...the information on the official website of Vivliostyle is only describing the application...

File: _posts/2021-11-17-meetup-2021-autumn-report.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+I also felt that the code reading that our old friend Kojima-san talked about was very important. One of the issues we've been facing in Vivliostyle lately is that the only committer for [Vivliostyle.js](https://github.com/vivliostyle/vivliostyle.js) is Founder Murakami. If we don't solve this problem, sustainability will be a concern. The only way to do this is to start by getting as many people as possible to read the source code, even if it seems a bit far-fetched. I realized that what we need now are measures to support that.
```
Comment (johanneswilm) [inline _posts/2021-11-17-meetup-2021-autumn-report.md:39]: Maybe change

> even if it seems a bit far-fetched.

to something like

> even if it seems like a strange place to start - we don't see any other way to do it.

Or something like that? I am not entirely sure what you are trying to say with that.

**Comment by johanneswilm** [comment]
I added two minor comments. overall this looks good!

File: _posts/2021-11-17-meetup-2021-autumn-report.md (hunk 4 above)
Comment (ogwata) [reply _posts/2021-11-17-meetup-2021-autumn-report.md]: “bit far-fetched” was indeed difficult to understand. I fixed it as follows.

> The only way to do this is to start by getting as many people as possible to read the source code, even if it seems like a **detour.**

Comment (johanneswilm) [reply _posts/2021-11-17-meetup-2021-autumn-report.md]: Ok. Sounds good

**Pull request merged**